src/components/Task/Modal/Form/LogForm/TaskModalLogForm.test.tsx

```diff
@@ -160,14 +150,7 @@ describe('TaskModalLogForm', () => {
       <LocalizationProvider dateAdapter={AdapterLuxon}>
```

Comment:
@canac do you have any ideas why this test isn't passing? I'm guessing it times out before the `onClose` function is called when saving. Someone added a wait time of `25000` so I'm guessing it has had problems in the past. Could waiting for useApiConstants be making it take longer?

Reply:
If you get add `debug` to the result that `render` returns and call `console.log(debug(undefined, Infinity))` after clicking save, you can see that there's a `disabled` attribute on the save button. My guess is that something about the form isn't valid.